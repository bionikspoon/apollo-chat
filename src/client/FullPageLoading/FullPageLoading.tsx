import * as React from 'react'

import { Spinner } from '@blueprintjs/core'
import { StyleSheet } from 'aphrodite'
import PageLayout from '../PageLayout'

export default function FullPageLoading() {
  return (
    <PageLayout css={styles.container}>
      <Spinner intent="primary" size={Spinner.SIZE_LARGE} />
    </PageLayout>
  )
}

const styles = StyleSheet.create({
  container: {
    justifyContent: 'center',
  },
})
