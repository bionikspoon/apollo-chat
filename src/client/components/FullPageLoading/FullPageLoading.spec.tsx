import { render } from 'enzyme'
import React from 'react'
import FullPageLoading from '.'

test('it matches snapshot', () => {
  const wrapper = render(<FullPageLoading />)
  expect(wrapper).toMatchSnapshot()
})
