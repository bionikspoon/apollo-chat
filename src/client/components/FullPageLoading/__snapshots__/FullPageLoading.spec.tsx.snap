// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<div
  class="bp3-ui-text bp3-dark container_ybok0l-o_O-container_agrn11"
>
  <div
    class="bp3-spinner bp3-intent-primary"
  >
    <div
      class="bp3-spinner-animation"
    >
      <svg
        height="100"
        stroke-width="4.00"
        viewBox="3.00 3.00 94.00 94.00"
        width="100"
      >
        <path
          class="bp3-spinner-track"
          d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90"
        >
          
        </path>
        <path
          class="bp3-spinner-head"
          d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90"
          pathLength="280"
          stroke-dasharray="280 280"
          stroke-dashoffset="210"
        >
          
        </path>
      </svg>
    </div>
  </div>
</div>
`;
